dev/com.ibm.ws.microprofile.config.1.3_fat/test-applications/serverXMLApp/src/com/ibm/ws/microprofile/config13/serverXML/web/ServerXMLBean.java

```diff
@@ -0,0 +1,56 @@
+/*******************************************************************************
+ * Copyright (c) 2018 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package com.ibm.ws.microprofile.config13.serverXML.web;
+
+import static org.junit.Assert.assertEquals;
+
+import javax.enterprise.context.RequestScoped;
+import javax.inject.Inject;
+
+import org.eclipse.microprofile.config.Config;
+import org.eclipse.microprofile.config.inject.ConfigProperty;
+import org.eclipse.microprofile.config.spi.ConfigBuilder;
+import org.eclipse.microprofile.config.spi.ConfigProviderResolver;
+
+@RequestScoped
+public class ServerXMLBean {
```

Comment:
Is this `@RequestScoped` to test dynamism?

Reply:
nope ... that would be way too cool